Dockerfile

```diff
@@ -1,15 +1,42 @@
-FROM node:16
-
-ENV NODE_ENV production
-ENV SERVER_HOST 0.0.0.0
-ENV PORT 4000
+#
+# Build
+#
+FROM node:16-slim AS builder
 
 WORKDIR /app
 
 # Install the project's dependencies, including dev dependencies.
 COPY package.json yarn.lock ./
+
+# python and build-essential are needed or the `yarn install --pure-lockfile`
+# fails while trying to invoke `/app/node_modules/node-pty`.
+RUN apt-get update -y && apt-get install -y python3 build-essential
 RUN yarn install --pure-lockfile --production=false
 
-COPY . .
+#
+# Install
+#
+FROM node:16-slim
+
+ARG olympia_uid=9500
```

Comment:
Could we pick another name than `olympia` ? `olympia` is the original name for `addons-server` specifically, so it's a bit weird to use it for a different repos.

Reply:
Made it `app_uid`. Hope that is generic enough.